fud/fud/stages/xilinx/emulation.py

```diff
@@ -44,7 +44,7 @@ def _define_steps(self, input_data, builder, config):
         )
 
         host_cpp = config["stages", self.name, "host"]
-        save_temps = bool(config["stages", self.name, "save_temps"])
+        save_temps = bool(config["stages", self.name, "save_temps"]) and config["stages", self.name, "save_temps"] == "true"
```

Comment:
Sorry to time travel back to this PR, but I'm curious about the need for this line—any chance it slipped in accidentally? (Seems like the main focus here is on the execution stage, not the emulation stage.)